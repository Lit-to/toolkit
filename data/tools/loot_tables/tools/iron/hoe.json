{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "weight": 1,
                    "name": "minecraft:iron_hoe",
                    "functions": [
                        {
                            "function": "set_count",
                            "count": 1
                        },
                        {
                            "function": "set_name",
                            "name": [
                                {
                                    "text": "鉄のクワ"
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}