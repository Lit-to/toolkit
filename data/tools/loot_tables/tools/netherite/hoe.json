{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "weight": 1,
                    "name": "minecraft:netherite_hoe",
                    "functions": [
                        {
                            "function": "set_count",
                            "count": 1
                        },
                        {
                            "function": "set_name",
                            "name": [
                                {
                                    "text": "ネザライトのクワ"
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}