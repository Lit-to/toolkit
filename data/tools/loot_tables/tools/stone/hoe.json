{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "weight": 1,
                    "name": "minecraft:stone_hoe",
                    "functions": [
                        {
                            "function": "set_count",
                            "count": 1
                        },
                        {
                            "function": "set_name",
                            "name": [
                                {
                                    "text": "石のクワ"
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}