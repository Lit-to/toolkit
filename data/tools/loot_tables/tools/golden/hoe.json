{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "weight": 1,
                    "name": "minecraft:golden_hoe",
                    "functions": [
                        {
                            "function": "set_count",
                            "count": 1
                        },
                        {
                            "function": "set_name",
                            "name": [
                                {
                                    "text": "金のクワ"
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}